server/routerlicious/packages/protocol-base/src/protocol.ts

```diff
@@ -58,16 +84,52 @@ export class ProtocolOpHandler {
         sendProposal: (key: string, value: any) => number,
     ) {
         this.term = term ?? 1;
-        this.quorum = new Quorum(
+        this._quorum = new Quorum(
             members,
             proposals,
             values,
             sendProposal,
         );
     }
 
+    public get attributes(): IDocumentAttributes {
+        return {
+            minimumSequenceNumber: this.minimumSequenceNumber,
+            sequenceNumber: this.sequenceNumber,
+            term: this.term,
+        };
+    }
+
+    setConnectionState(connected: boolean, clientId: string | undefined) {
+        this._quorum.setConnectionState(connected, clientId);
+    }
+
+    snapshot(): IQuorumSnapshot {
+        return this._quorum.snapshot();
+    }
+
     public close() {
-        this.quorum.close();
+        this._quorum.close();
+    }
+
+    public processMessageInQuorum(message: ISequencedDocumentMessage, local: boolean): IProcessMessageResult {
```

Comment:
Other than this function, looks good. Please see my comment in #10412 - I believe that's wrong way to go about it. To some extend your finding confirms that different entities might have different needs RE protocol implementation (though I'm a bit surprised that Scribe is in this domain - I worry that it points to a potential bug somewhere).